jdaviz/app.py

```diff
@@ -265,13 +265,22 @@ def load_data(self, file_obj, parser_reference=None, **kwargs):
         ----------
         file_obj : str or file-like
             File object for the data to be loaded.
+
+        parser_reference
+
+        skip_checks : bool
```

Comment:
This is necessary to bypass default check so I can pass in things like `filename.fits[ext]`.

Reply:
Tom R and I talked about this -- Might be better to not add this new keyword and use `**kwargs` to pass `ext` around... There are more details inside my head and they will become clear in a subsequent commit.